runtime/buildtools.mk

```diff
@@ -35,6 +35,37 @@ BUILD_ID      ?= 000000
 OS            := $(shell uname)
 TRC_THRESHOLD ?= 1
 FREEMARKER_JAR ?= $(CURDIR)/buildtools/freemarker.jar
+CMAKE_ARGS :=
+
+# We grab the C/C++ compilers detected by autoconf or provided by user, not
+# the CC/CXX variables defined by # the makefiles, which potentitally include
```

Comment:
There is an extra comment character in the middle of this line.

Reply:
Fixed